package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "s47_list_of_inspirational_movies",
```

Comment:
Consider adding more information to the package.json file, such as the author's name, a more descriptive project name, and any dependencies or devDependencies. [medium]